.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: ruby
 rvm:
-  - 2.3.4
-  - 2.4.1
+  - 2.3.8
```

Comment:
I'm not too familiar with the travis syntax, but could we leave off the patch version to get the latest?  Something like this:

```yaml
rvm:
  - 2.3
  - 2.4
  - 2.5
  - 2.6
```

Reply:
Excellent question! I'm not sure, either. Let me look at the docs.

Reply:
Ok, yes it appears we can do that: https://docs.travis-ci.com/user/languages/ruby/

I'll make that change before merging.